test/integration/webhook_test.go

```diff
@@ -317,27 +317,32 @@ func TestWebhookGitHubPushWithImageStream(t *testing.T) {
 	}
 	defer watch.Stop()
 
-	for _, s := range []string{
-		"/oapi/v1/namespaces/" + testutil.Namespace() + "/buildconfigs/pushbuild/webhooks/secret101/github",
-	} {
+	s := "/oapi/v1/namespaces/" + testutil.Namespace() + "/buildconfigs/pushbuild/webhooks/secret101/github"
```

Comment:
did we just overlook adding gitlab and bitbucket versions here, or have I forgotten the reason gitlab and bitbucket do not apply here?

Reply:
it's just not relevant to what we're trying to test i don't think...one is sufficient.  We just want to see that webhook triggered builds get imagestreams resolved.